tests/contrib/logging/test_logging.py

```diff
@@ -1,13 +1,22 @@
 import logging
+import sys
 
 import ddtrace
-from ddtrace.constants import ENV_KEY, VERSION_KEY
 from ddtrace.compat import StringIO
+from ddtrace.constants import ENV_KEY, VERSION_KEY
 from ddtrace.contrib.logging import patch, unpatch
+from ddtrace.contrib.logging.patch import RECORD_ATTR_TRACE_ID, RECORD_ATTR_SPAN_ID
 from ddtrace.vendor import wrapt
 
 from ... import TracerTestCase
 
+PY2 = sys.version_info[0] == 2
+
+if PY2:
```

Comment:
We can use `six.string_types` here available from `ddtrace.vendor.six` 🙂

Reply:
Thanks for the pointer, I was trying to use code I found in other parts of the codebase 😛